tsconfig.common.json

```diff
@@ -25,6 +27,6 @@
     "skipLibCheck": true,
     "suppressImplicitAnyIndexErrors": true
   },
-  "exclude": ["lib", "node_modules", "**/*.test.*", "**/*.spec.*"],
+  "exclude": ["lib", "node_modules"],
```

Comment:
This was actually the KEY problem with Jest. Babel typescript-preset leverages this file and therefore was ignoring test and spec files. 🤦

Reply:
Makes sense. Good find.